src/tests/test_parser.rs

```diff
@@ -197,9 +198,11 @@ fn equals_separates_long_flag() -> TestResult {
 
 #[test]
 fn let_env_expressions() -> TestResult {
-    run_test(
-        r#"let-env PATH = if (env | any? name == VENV_OLD_PATH) { $env.VENV_OLD_PATH } else { $env.PATH }; echo done"#,
-        "done",
+    let env = HashMap::from([("VENV_OLD_PATH", "Foobar")]);
+    run_test_with_env(
+        r#"let-env Path = if (env | any? name == VENV_OLD_PATH) { $env.VENV_OLD_PATH } else { $env.Path }; echo $env.Path"#,
```

Comment:
Unfortunately, this is not always the case on Windows. Sometimes it's `$env.PATH` and other times it's `$env.Path` depending on how you launched nushell.exe.

Reply:
Ah, quite right. I thought the else would only expand if the condition wasn't true, but that's not the case. I'll fix it.

Reply:
OK, explicitly added `Path` to the test environment before running the `let-env` command now.

Reply:
you're circumventing the environment and making your own. I guess that's ok?

Reply:
Yeah the reasoning for setting some expected env vars for the process is that we don't have to make assumptions.